internal/services/keyvault/key_vault_key_resource.go

```diff
@@ -791,10 +798,8 @@ func flattenKeyVaultKeyRotationPolicy(input keyvault.KeyRotationPolicy) []interf
 		}
```

Comment:
if the `policy` block is being set into the state then all fields need to be set all of the time, fwiw